Sentry.xcodeproj/project.pbxproj

```diff
@@ -1583,7 +1583,9 @@
 				63AA75921EB8AEDB00D153DE /* Tests */,
 				7D826E3C2390840E00EED93D /* Utils */,
 			);
+			indentWidth = 4;
```

Comment:
My Xcode is set to indent with 2 spaces by default, so this overrides that for the project, the sources and the samples.

Reply:
Hmm, @armcknight already pointed out that we should maybe switch to XcodeGen to avoid such issues. Can you please not merge this change?

Reply:
If I remove this, then all my code will be indented by 2 spaces unless I change my Xcode's defaults (which is possible, but surely I'm not the only one with defaults set to 2). Switching to XcodeGen is probably not something that's going to happen very soon?

Reply:
> Switching to XcodeGen is probably not something that's going to happen very soon

No, not really. 

Ah sorry, I misunderstood your comment. It's fine to merge this. You could even open up another PR, so this gets merged faster.

Reply:
I'll remove it from here, since I've created #1934